### PR TITLE
Convert `pF` commands to the rzshell

### PR DESCRIPTION
**Pull request by XVilka**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Convert `pF` commands to rzshell

**Test plan**

CI is green.

**Closing issues**

Partially addresses https://github.com/rizinorg/rizin/issues/1590

### CODE REVIEW

File: librz/core/cmd_descs/cmd_print.yaml [hunk 1]
```diff
@@ -345,6 +345,44 @@ commands:
             summary: Summarize current block
             cname: cmd_disassemble_summarize_block
             args: []
+  - name: pF
+    summary: Print parsed ASN.1, PKCS, X590, DER, etc formats
```
Comment (wargio) [inline librz/core/cmd_descs/cmd_print.yaml]:
```suggestion
    summary: Print parsed ASN.1, PKCS, X509, ProtoBuf, etc.. formats
```

File: librz/core/cmd_descs/cmd_print.yaml [hunk 2]
```diff
@@ -345,6 +345,44 @@ commands:
...
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
+      - name: pFA
+        summary: Decode Android Binary XML from current block
+        cname: cmd_print_axml
+        args:
+          - name: len
+            type: RZ_CMD_ARG_TYPE_NUM
+            optional: true
```
Comment (wargio) [inline librz/core/cmd_descs/cmd_print.yaml]: in the command you do not supply this value, but just use the `core->block[size]` variables. i guess we can remove the args.

**Review by wargio — CHANGES_REQUESTED** [review]

File: librz/core/cmd_descs/cmd_print.yaml (hunk 2 above)
Comment (ret2libc) [reply librz/core/cmd_descs/cmd_print.yaml]: yes, same for all the others please

File: librz/core/cmd/cmd_print.c [hunk 3]
```diff
@@ -6599,3 +6511,72 @@ RZ_IPI RzCmdStatus rz_print_byte_bitstream_handler(RzCore *core, int argc, const
...
+		return RZ_CMD_STATUS_ERROR;
+	}
+	RzStrBuf *sb = rz_strbuf_new("");
+	rz_x509_certificate_dump(x509, NULL, sb);
+	char *res = rz_strbuf_drain(sb);
+	if (res) {
+		rz_cons_printf("%s\n", res);
+		free(res);
+	}
+	rz_x509_free_certificate(x509);
```
Comment (ret2libc) [inline librz/core/cmd/cmd_print.c]: weird naming convention here :(

Comment (wargio) [reply librz/core/cmd/cmd_print.c]: I will fix this today

Comment (wargio) [reply librz/core/cmd/cmd_print.c]: My MR should be merged before this one: https://github.com/rizinorg/rizin/pull/3077

**Review by XVilka — COMMENTED** [review]
~~[XX] db/cmd/cmd_pFb pFb.test simple protobuf (verbose)~~ Resolved

<details>
  <summary markdown="span">click me to expand</summary>

```
[XX] db/cmd/cmd_pFb pFb.test simple protobuf (verbose)
RZ_NOPLUGINS=1 /usr/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'wx 0a 2e 0a 07 56 69 6e 63 65 6e 74 10 09 1a 10 56 69 6e 63 65 6e 74 40 74 65 73 74 2e 63 6f 6d 22 0f 0a 0b 31 35 30 31 31 31 31 31 31 31 31 10 02
pFbv
' =
-- stdout
--- expected
+++ actual
@@ -1,9 +1,33 @@
-1 [LEN_DELIM]   {
-    1 [LEN_DELIM]  : "Vincent"
-    2 [VARINT]     : 9 | 9
-    3 [LEN_DELIM]  : "Vincent@test.com"
-    4 [LEN_DELIM]   {
-        1 [LEN_DELIM]  : "15011111111"
-        2 [VARINT]     : 2 | 2
-    }
-}
+Usage: p[=68abcdDfiImrstuxz] [arg|len] [@addr]  
+| p[b|B|xb] [len] ([S])   bindump N bits skipping S bytes
+| p[iI][df] [len]         print N ops/bytes (f=func) (see pi? and pdq)
+| p[kK] [len]             print key in randomart (K is for mosaic)
+| p-[?][jh] [mode]        bar|json|histogram blocks (mode: e?search.in)
+| p2 [len]                8x8 2bpp-tiles
+| p6[de] [len]            base64 decode/encode
+| p8[?][j] [len]          8bit hexpair list of bytes
+| p=[?][bep] [N] [L] [b]  show entropy/printable chars/chars bars
+| pa[edD] [arg]           pa:assemble  pa[dD]:disasm or pae: esil from hex
+| pA[n_ops]               show n_ops address and type
+| pb[?] [n]               bitstream of N bits
+| pB[?] [n]               bitstream of N bytes
+| pc[?][p] [len]          output C (or python) format
+| pC[aAcdDxw] [rows]      print disassembly in columns (see hex.cols and pdq)
+| pd[?] [sz] [a] [b]      disassemble N opcodes (pd) or N bytes (pD)
+| pf[?][.nam] [fmt]       print formatted data (pf.name, pf.name $<expr>)
+| pF[?][apx]              print asn1, pkcs7 or x509
+| pg[?][x y w h] [cmd]    create new visual gadget or print it (see pg? for details)
+| ph[?][=|hash] ([len])   calculate hash for a block
+| pi[?][bdefrj] [num]     print instructions
+| pI[?][iI][df] [len]     print N instructions/bytes (f=func)
+| pj[?] [len]             print as indented JSON
+| pm[?] [magic]           print libmagic data (see pm? and /m?)
+| po[?] hex               print operation applied to block (see po?)
+| pp[?][sz] [len]         print patterns, see pp? for more help
+| pr[?][glx] [len]        print N raw bytes (in lines or hexblocks, 'g'unzip)
+| ps[?][pwz] [len]        print pascal/wide/zero-terminated strings
+| pt[?][dn] [len]         print different timestamps
+| pu[?][w] [len]          print N url encoded bytes (w=wide)
+| pv[?][jh] [mode]        show variable/pointer/value in memory
+| px[?][owq] [len]        hexdump of N bytes (o=octal, w=32bit, q=64bit)
+| pz[?] [len]             print zoom view (see pz? for help)



[XX] db/cmd/cmd_pFb pFb.test simple protobuf
RZ_NOPLUGINS=1 /usr/bin/rizin -escr.utf8=0 -escr.color=0 -escr.interactive=0 -N -Qc 'wx 0a 2e 0a 07 56 69 6e 63 65 6e 74 10 09 1a 10 56 69 6e 63 65 6e 74 40 74 65 73 74 2e 63 6f 6d 22 0f 0a 0b 31 35 30 31 31 31 31 31 31 31 31 10 02
pFb
' =
-- stdout
--- expected
+++ actual
@@ -1,9 +1,9 @@
-1 {
-    1: "Vincent"
-    2: 9 | 9
-    3: "Vincent@test.com"
-    4 {
-        1: "15011111111"
-        2: 2 | 2
+1 [LEN_DELIM]   {
+    1 [LEN_DELIM]  : "Vincent"
+    2 [VARINT]     : 9 | 9
+    3 [LEN_DELIM]  : "Vincent@test.com"
+    4 [LEN_DELIM]   {
+        1 [LEN_DELIM]  : "15011111111"
+        2 [VARINT]     : 2 | 2
     }
 }
```
</details>

**Review by XVilka — COMMENTED** [review]
LGTM

**Pull request merged**